webapi/src/test/scala/org/knora/webapi/responders/v1/ResourcesResponderV1Spec.scala

```diff
@@ -905,7 +890,9 @@ class ResourcesResponderV1Spec extends CoreSpec(ResourcesResponderV1Spec.config)
             actorUnderTest ! ResourceContextGetRequestV1(iri = "http://data.knora.org/c5058f3a", resinfo = true, userProfile = SharedAdminTestData.incunabulaProjectAdminUser)
 
             expectMsgPF(timeout) {
-                case response: ResourceContextResponseV1 => compareResourceCompoundContextResponses(received = response, expected = ResourcesResponderV1SpecContextData.expectedBookResourceContextResponse)
+                case response: ResourceContextResponseV1 =>
+                    val responseAsJson = response.toJsValue
+                    assert(responseAsJson == ResourcesResponderV1SpecContextData.expectedBookResourceContextResponse, "book context response did not match")
```

Comment:
Yes, that is absolutely correct. I justed wanted to make sure that this is what we want.